sdk/storage/ustorageclient/samples/blob_append_bytes/main.c

```diff
@@ -82,4 +82,6 @@ void main(void)
 
     // end sample
     (void)printf("\n********** Sample '%s' FINISH **********\r\n", AZSTORAGE_CONFIG_SAMPLE_NAME);
```

Comment:
what is the carrage-return doing here? printf will "do the right thing" on windows even without it? Did you reopen stdout in binary mode someplace?